mock/storage_reads.go

```diff
@@ -98,6 +101,38 @@ func (rs *GroupResultSet) Err() error {
 	return rs.ErrFunc()
 }
 
+type FloatArrayCursor struct {
+	CloseFunc func()
+	Errfunc   func() error
```

Comment:
nit, the `f` in `Errfunc` could be uppercase to be consistent with the other member names.

Reply:
This is duplicated from `IntegerArrayCursor` which was written by somebody else so I was thinking I would keep it the same for consistency.